src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,40 @@
+package duke;
+import java.time.LocalDateTime;
```

Comment:
Consider adding a line break between the packaging and import statements, perhaps